web/src/components/entry-dialog/useEntryForm.tsx

```diff
@@ -69,6 +73,32 @@ const useEntryForm = ({ editEntry, date }: useEntryProps) => {
 
   const form = useForm<EntryFormSchema>({ defaultValues });
 
+  const { watch } = form;
+  const dateWatch = dayjs(watch("date")).locale(dayjs.locale());
+
+  const queryDate = dateWatch.format("YYYY-MM-DD");
+  const { data, loading: wdLoading } = useQuery(FindWorkdaysDocument, {
+    variables: { start: queryDate, end: queryDate },
+  });
+  const setDuration = useCallback((hours: string) => form.setValue("duration", hours), [form]);
+
+  useEffect(() => {
+    if (!wdLoading && userPrefersSetRemainingHours) {
+      const totalDuration = totalDurationOfEntries(
+        data?.findWorkdays.length === 1 ? data.findWorkdays[0].entries : [],
+      );
+      const workDayFullHours = dayjs.duration(7.5, "hour");
+      const remainingHours =
+        workDayFullHours > totalDuration
+          ? workDayFullHours.subtract(totalDuration)
+          : dayjs.duration(0);
+      const remainingHoursFormatted = roundToFullMinutes(remainingHours).asHours().toString();
+      setDuration(remainingHoursFormatted);
```

Comment:
IDK if reset [(reset)](https://react-hook-form.com/docs/useform/reset) would be better option to set the duration. I tried using reset quickly but it bugged somehow -> maybe I didn't know how to use it. Using reset we can set the default values of the form, which means that when user wants to clear the form manually, the duration would reset to that value.

Reply:
Using reset now it should work as excpected